internal/table/table.go

```diff
@@ -133,51 +140,120 @@ func rescaleColumns(colWidths []int, targetTotal int) {
 	colWidths[len(colWidths)-1] += targetTotal - mathutils.Total(colWidths...)
 }
 
+func getCroppedText(text string, maxLen int) []entry {
```

Comment:
This is not accounting for any existing linebreaks within the text.

eg.

```
getCroppedText("foo\nbar", 4)
```

Should result in 

```
[]entry{{"foo", 3}, {"bar", 3}}
```

Whereas your current implementation would produce

```
[]entry{{"foo\n", 4}, {"bar", 3}}
```